examples/src/models/chat/integration_tencent_hunyuan.ts

```diff
@@ -0,0 +1,115 @@
+import { ChatTencentHunyuan } from "@langchain/community/chat_models/tencent_hunyuan";
+import { HumanMessage } from "@langchain/core/messages";
+import type { LLMResult } from "@langchain/core/outputs";
+
+// in nodejs environment
+import { sign } from "@langchain/community/utils/tencent_hunyuan/index";
+
+// in browser environment
+// import { sign } from "@langchain/community/utils/tencent_hunyuan/web";
```

Comment:
Up to you, but could also just make separate chat model and embedding entrypoints for web vs. Node

Reply:
For example: https://github.com/langchain-ai/langchainjs/tree/main/libs/langchain-community/src/chat_models/bedrock

Reply:
Hi @jacoblee93 , I have adjusted according to the bedrock example, making separate entrypoints for web & node. In the newest commit, `sign` method is not exported, users only need to use the right entrypoint. Meanwhile, I have adjusted the entrypoint name for node from `/chat_models/tencent_hunyuan/index` to `/chat_models/tencent_hunyuan` to make it more intuitive, the entrypoint for web changes to `/chat_models/tencent_hunyuan/web` accordingly.